# AA200 – Problem Set 5, Problem 2
## Elliptical Wing — Lifting-Line Theory

**Given:** AR = 5, $m_0 = 5.7$ /rad, $V = 45$ m/s, $W/S = 1000$ N/m², $b = 10$ m, sea level ($\rho = 1.225$ kg/m³), level flight ($L=W$).

---
## 1. Lift Coefficient

In level flight $L = W$, so:

$$C_L = \frac{W/S}{\tfrac{1}{2}\rho V^2} = \frac{1000}{\tfrac{1}{2}\times 1.225 \times 45^2}$$

In [ ]:
import numpy as np

AR = 5; m0 = 5.7; V = 45.0; WoS = 1000.0; rho = 1.225; b = 10.0
S = b**2 / AR

CL = WoS / (0.5 * rho * V**2)
print(f'S  = {S:.2f} m²')
print(f'CL = {CL:.4f}')

S  = 20.00 m²
CL = 0.8062


---
## 2. Sectional and Total Lift & Drag Coefficients

Here $m_0 = C_{L_\alpha} = 5.7$ /rad is the **finite-wing** lift-curve slope. We invert the lifting-line relation to recover the sectional slope:

$$m_0 = \frac{c_{\ell_\alpha}}{1 + c_{\ell_\alpha}/(\pi AR)} \implies c_{\ell_\alpha} = \frac{m_0}{1 - m_0/(\pi AR)}$$

For an elliptical wing ($e=1$):

$$c_\ell(y) = C_L \quad \text{(uniform spanwise)}$$

$$C_{D_i} = \frac{C_L^2}{\pi AR}$$

In [ ]:
# m0 is the finite-wing lift-curve slope
CLa = m0  # = C_L_alpha = 5.7 /rad
cla = m0 / (1 - m0 / (np.pi * AR))  # sectional lift-curve slope
CDi = CL**2 / (np.pi * AR)

print(f'c_l_alpha (sectional) = {cla:.4f} /rad')
print(f'C_L_alpha (finite wing) = {CLa:.4f} /rad')
print(f'c_ell = C_L = {CL:.4f}   (uniform spanwise)')
print(f'C_Di  = {CDi:.4f}')

c_l_alpha (sectional) = 8.9526 /rad
C_L_alpha (finite wing) = 5.7000 /rad
c_ell = C_L = 0.8062   (uniform spanwise)
C_Di  = 0.0414


---
## 3. Angles of Attack

For an elliptical wing both angles are **constant spanwise**.

The induced angle:
$$\alpha_i = \frac{C_L}{\pi AR}$$

The absolute (sectional) angle of attack uses the **sectional** slope:
$$\alpha_0 = \frac{c_\ell}{c_{\ell_\alpha}}$$

In [ ]:
ai_rad = CL / (np.pi * AR)
a0_rad = CL / cla

print(f'alpha_i = {np.degrees(ai_rad):.4f} deg  ({ai_rad:.4f} rad)')
print(f'alpha_0 = {np.degrees(a0_rad):.4f} deg  ({a0_rad:.4f} rad)')

alpha_i = 2.9408 deg  (0.0513 rad)
alpha_0 = 5.1588 deg  (0.0900 rad)


---
## 4. Spanwise Plot

In [ ]:
ai_rad = CL / (np.pi * AR)
a0_rad = CL / cla

print(f'alpha_i = {np.degrees(ai_rad):.4f} deg  ({ai_rad:.4f} rad)')
print(f'alpha_0 = {np.degrees(a0_rad):.4f} deg  ({a0_rad:.4f} rad)')

alpha_i = 2.9408 deg  (0.0513 rad)
alpha_0 = 5.1588 deg  (0.0900 rad)


---
## 5. Power Required to Overcome Induced Drag

$$P = D_i \cdot V = \tfrac{1}{2}\rho V^2 S \cdot C_{D_i} \cdot V$$

In [ ]:
Di = 0.5 * rho * V**2 * S * CDi
Power = Di * V

print(f'Induced drag  Di = {Di:.2f} N')
print(f'Power required P = {Power:.1f} W  ({Power/1000:.2f} kW)')

Induced drag  Di = 1026.55 N
Power required P = 46194.6 W  (46.19 kW)


---
## Summary

| Quantity | Value |
|---|---|
| $S$ | 20.00 m² |
| $C_L = c_\ell$ | 0.8062 |
| $m_0 = C_{L_\alpha}$ | 5.7 /rad |
| $c_{\ell_\alpha}$ | 8.95 /rad |
| $C_{D_i}$ | 0.0414 |
| $\alpha_i$ | 2.94° (constant spanwise) |
| $\alpha_0$ | 5.16° (constant spanwise) |
| $D_i$ | 1026.6 N |
| Power | 46.2 kW |